integreat_cms/cms/templates/pois/poi_list_row.html

```diff
@@ -32,9 +30,9 @@
     {% endif %}
 	<td class="pr-2 lang-grid">
         {% spaceless %}
-            {% for other_language in languages %}
-                <a href="{% url 'edit_poi' poi_id=poi.id region_slug=region.slug language_slug=other_language.slug %}">
-                    {% get_translation poi other_language.slug as other_translation %}
+            {% get_poi_translations poi as translations %}
+            {% for other_translation in translation %}
+                <a href="{% url 'edit_poi' poi_id=poi.id region_slug=region.slug language_slug=other_translation.language.slug %}">
```

Comment:
The problem with this approach is that languages with missing translations are skipped. Also the order might differ from the languages in the grid, so you would have to make sure these are always identical.